contrastsecurity/manifest.json

```diff
@@ -0,0 +1,29 @@
+{
+  "display_name": "contrastsecurity",
+  "maintainer": "kristiana.mitchell@contrastsecurity.com",
+  "manifest_version": "1.0.0",
+  "name": "contrastsecurity",
+  "metric_prefix": "contrastsecurity.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "",
```

Comment:
Hello! This needs to be non-null. Just put a short description of the integration here. Reference some of the other `extras` integrations for inspiration. :)

Reply:
Please, add a short description of the integration here.